### ISSUE TITLE
After some directory changes the mouse click to images does not work anymore

### ISSUE THREAD

**Issue by mowgli** [BestImageViewer/geeqie#847]
### ISSUE TYPE
* Bug Report

### GEEQIE VERSION
master (f34ea0700048)

GTK3

The following is always needed for me to use my mouse in gtk3; so this bug might be connected…
export GDK_CORE_DEVICE_EVENTS=1

### OS / DISTRIBUTION
Devuan (Debian) Sid

### SUMMARY
Some times, after several directory changes *without changing the pictures in dirs in between* the mouse stops working for changing the pictures. Although, everything else work; going to fullscreen by double click, wheel, changing dirs, … work well. Only clicking to the image in the file list doesn't.

### STEPS TO REPRODUCE
I have no real idea. Sometimes it works, sometimes it doesn't.

**Comment by mowgli** [comment]
An unset GDK_CORE_DEVICE_EVENTS doesn't matter.

**Comment by tomaszg7** [comment]
https://github.com/BestImageViewer/geeqie/issues/769

**Comment by mowgli** [comment]
Ah, thanks for the hint. However, it does not show, where the error is located.

I even tried to bisect but I do not find a working version to start with. I think, I have a pretty good way to reporduce it. Even if not 100% sure it is a start.

**Comment by tomaszg7** [comment]
Yeah, I didn't know either. The bug might not even be inside Geeqie, might be e.g. some change in gtk or other lib. 

If I find more time, I'll probably try to do something about it, probably starting with sprinkling the code with excess of debug messages to pinpoint where exactly the problem happens.

**Comment by mowgli** [comment]
I tested with gtk2 now.
The bug are definitively related to gtk3.

Not only that, geeqie is much more faster with gth2 than with gtk3!

But there is one, maybe related issue. In gtk2 (not in gtk3) some directory changes are not updated in the dir- and file- lists. One have to scroll in one of them to get them both updated. Maybe this is how the bug is manifested in gtk2.
